packages/react-components-pixi-7/package.json

```diff
@@ -41,53 +41,49 @@
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
-    "@pixi/react-fiber": "*",
-    "@pixi/react-invariant": "*",
-    "@pixi/react-modular": "*",
-    "@pixi/math": "^7.0.0",
     "@pixi/app": "^7.0.0",
-    "@pixi/ticker": "^7.0.0",
+    "@pixi/constants": "^7.0.0",
+    "@pixi/core": "^7.0.0",
     "@pixi/display": "^7.0.0",
     "@pixi/events": "^7.1.1",
-    "@pixi/core": "^7.0.0",
-    "@pixi/sprite-animated": "^7.0.0",
-    "@pixi/text-bitmap": "^7.0.0",
     "@pixi/graphics": "^7.0.0",
+    "@pixi/math": "^7.0.0",
     "@pixi/mesh-extras": "^7.0.0",
     "@pixi/particle-container": "^7.0.0",
-    "@pixi/constants": "^7.0.0",
+    "@pixi/react-fiber": "*",
+    "@pixi/react-invariant": "*",
+    "@pixi/react-modular": "*",
     "@pixi/sprite": "^7.0.0",
-    "@pixi/text": "^7.0.0",
+    "@pixi/sprite-animated": "^7.0.0",
     "@pixi/sprite-tiling": "^7.0.0",
+    "@pixi/text": "^7.0.0",
+    "@pixi/text-bitmap": "^7.0.0",
+    "@pixi/ticker": "^7.0.0",
     "@testing-library/react": "^13.4.0",
-    "canvas": "^2.11.0",
     "jest-webgl-canvas-mock": "^0.2.3",
-    "prop-types": "^15.8.1",
```

Comment:
This is used in the `Stage` component isn't it?

Reply:
Yeah it is, that is why it was moved to a dependency!

Reply:
Sorry, missed that 👍